distributions/raspijamulus.sh

```diff
@@ -5,7 +5,7 @@ OPUS="opus-1.3.1"
 NCORES=$(nproc)
 
 # install required packages
-pkgs='alsamixergui build-essential qt5-default libasound2-dev cmake libglib2.0-dev'
+pkgs='alsamixergui build-essential qt5-qmake qtdeclarative5-dev qttools5-dev-tools libjack-jackd2-dev cmake libglib2.0-dev'
```

Comment:
I have just tried this script on a Pi with fresh install of Raspbian 10 (which has Qt 5.11.3). I found that it still needs `qt5-default` and `libasound2-dev`:
* without `libasound2-dev`, the build of `jack2` fails
* without `qt5-default`, the build of Jamulus fails.
I know that `qttools5-dev-tools` is needed for Qt >= 5.12 to provide `lrelease`.
And I have just seen your comment that `qt5-default` is not in Raspbian 11 (I was just about to try it myself). Interesting... it would be good if we could make this script work on both OS versions.

Reply:
> * without `libasound2-dev`, the build of `jack2` fails

I have just found this is also true on Raspbian 11, so `libasound2-dev` must still be included.

Reply:
Interesting. It did compile fine on my Raspbian. But maybe I had this package `libasound2-dev` was already installed. I'll do some more tests and update the PR, soon.

Reply:
Adding `libasound2-dev` is no problem because it is available. But since qt5-default is not available and cannot be installed, running the raspijamulus script will always ask `Do you want to install missing packages?` but if you answer with yes you'll get
```
Package qt5-default is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'qt5-default' has no installation candidate
```
The idea was that the script simply runs without interaction as soon as it was once installing everything.

Reply:
Ups... I just noticed another thing which was incorrect (it's a very long time ago when I was initially writing this script): Since we checkout our own version of jack2 and compile it, we also use the headers and libraries from that location and do not use the system jack2-dev files. So, I also removed the `libjack-jackd2-dev` again. Sorry for the inconvenience...

Reply:
One thing I have found when running this script on a fresh OS (same on both 10 and 11), is that when Jamulus compilation is finished, and the script runs Jamulus, it complains that it cannot connect to or start a jack server, and suggests starting jackd manually.

Reply:
You need to have an USB audio card connected when you start the script. It takes the first device it finds `(ADEVICE=$(aplay -l|grep "USB Audio"|tail -1|cut -d' ' -f3))` and tells you as an output what device it found and wants to use: `echo "Using USB audio device: ${ADEVICE}"`. I assume that in your case no audio device was found and therefore the jack daemon could not be started.